src/components/table/Table.js

```diff
@@ -1060,7 +1066,8 @@ class Table extends Component {
         ref={(c) => {
           if (c) {
             const keyProp = item[keyProperty];
-            this.rowRefs[keyProp] = c.wrappedInstance;
+
+            this.rowRefs[keyProp] = c;
```

Comment:
Can you pls refactor a bit and put a meaningful name instead of just c? Thank you

Reply:
Also was wondering what do you think about the check from codacy -> https://app.codacy.com/gh/metasfresh/metasfresh-webui-frontend/pullRequest?prid=5129401. Should we remove that ? we could forcely set the parseInt for some to fix it but code would look ugly..  I believe we could ditch that  Generic Object Injection Sink check. Let me know your thoughts

Reply:
Yeah, let's remove that. It's a dumb and completely unnecessary check.

Reply:
Do we really need a meaningful name in `ref` function ? We all know how it works, and changing this in one place would mean we have to change it everywhere to stay consistent. `c` or `ref` is what we're using all across the app.

Reply:
Would be nice to have it -> https://github.com/ryanmcdermott/clean-code-javascript#variables (Avoid Mental Mapping)

Reply:
^^ only referring to (c)

Reply:
Yeah but then again we shouldn't get crazy about it. `c` in ref is always a component.